### PR TITLE
20 - Comment Model

### PR DESCRIPTION
**Pull request by mar7in91Wolox**
# Summary
- I added the _Models/Database/Comment.cs_ with the fields _Id_, _CommentString_, _User_ and _MovieId_.
- I added the field _Comments_ into _Models/Database/Movie.cs_. That defines a relation one-to-many between Movie and Comment.

# Trello Card
https://trello.com/c/v0zbvMuw

### CODE REVIEW

File: training-net/src/Models/Database/Comment.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+namespace training_net.Models
+{
+    public class Comment
+    {
+        public int Id { get; set; }
+        public string CommentString { get; set;}
+        public string User { get; set;}
+        public Movie Movie { get; set;}
+    }
+}
```
Comment (truccomarcos) [inline training-net/src/Models/Database/Comment.cs]: Don't forget to add an empty line at EOF

File: training-net/src/Models/Database/Comment.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace training_net.Models
+{
+    public class Comment
+    {
+        public int Id { get; set; }
+        public string CommentString { get; set;}
+        public string User { get; set;}
```
Comment (truccomarcos) [inline training-net/src/Models/Database/Comment.cs]: Did you change this to be an actual User instance?

File: training-net/src/Models/Database/Comment.cs (hunk 1 above)
Comment (mar7in91Wolox) [reply training-net/src/Models/Database/Comment.cs]: Done!

File: training-net/src/Models/Database/Comment.cs (hunk 2 above)
Comment (mar7in91Wolox) [reply training-net/src/Models/Database/Comment.cs]: @truccomarcos Could you check the last changes before I do the migration?

**Pull request merged**